src/clientSideScene/sceneEntities.ts

```diff
@@ -343,11 +343,8 @@ export class SceneEntities {
       pathToNode: sketchPathToNode,
     }
     const dummy = new Mesh()
-    dummy.position.set(
-      sketchGroup.position[0],
-      sketchGroup.position[1],
-      sketchGroup.position[2]
-    )
+    // TODO: When we actually have sketch positions and rotations we can use them here.
```

Comment:
I removed these because when we do this I don't want to be tied to what was essentailly dead code in the rust code, we can implement how we want then.